src/mokka/src/main/java/pl/hycom/mokka/emulator/logs/LogsController.java

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
`getLogs` was modified by adding `name` request parameter.
Please cover this in `LogsControllerIntegrationTest`